### PR TITLE
release for draft info and other fixes

### PR DESCRIPTION
**Pull request by davidgamero**
# Description

### Added

- The new `draft info` command from #157 prints supported language and field information in json format.
- An integration test was added for the installation shell script to better ensure that the script works as expected.

### Fixed

- File path output for root locations had a bug with string-formatted paths. The `path.Join` method has been substituted to fix this.

### Changed

- Remaining uses of the `viper` library have been migrated to `gopkg.in/yaml.v3` for consistency.
- Unused files in the `web` package have been removed.
- Minor reorganization across the `config` and `addons` packages to reduce the number of exported functions and types.

## Type of change

Release

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by qpetraroia — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,22 @@
 # Change Log
+All notable changes to this project will be documented in this file.
```
Comment (Tatsinnit) [inline CHANGELOG.md:2]: 💡 Thanks for adding me to the review. 📓 Please note: all this is scaffolding text from the action, you can safely remove it and only keep `##` part. 

**Samples below:** Also some of the simple samples are here: (If you want to see sample and adapt that)

* https://github.com/Azure/vscode-aks-tools/blob/master/CHANGELOG.md 
* https://github.com/Azure/aks-periscope/blob/master/CHANGELOG.md

**Review by Tatsinnit — DISMISSED** [review]
💡 🎊 Thanks for adding me to this review, I have added something which we can remove, please feel free to take a look and decide. Thanks.

File: CHANGELOG.md (hunk 1 above)
Comment (davidgamero) [reply CHANGELOG.md]: thank you i've removed it

**Review by Tatsinnit — APPROVED** [review]
🎉❤️☕️

**Pull request merged**